### ISSUE TITLE
[error] Error: EISDIR: illegal operation on a directory, read

### ISSUE THREAD

**Issue by StephenFluin** [angular/angular-cli#16858]
# 🐞 Bug report

### Command (mark with an `x`)

- [x] build
- [x] serve

### Is this a regression?
yes, wasn't present in 8.x

### Description

`ng serve` fails with the error:
```
An unhandled exception occurred: EISDIR: illegal operation on a directory, read
See "/tmp/ng-cDbmgA/angular-errors.log" for further details.
```

and this file contains
```
[error] Error: EISDIR: illegal operation on a directory, read
    at Object.readSync (fs.js:506:3)
    at tryReadSync (fs.js:331:20)
    at Object.readFileSync (fs.js:368:19)
    at loadTsLibFiles (/usr/local/google/home/stephenfluin/workspace/test/devfest/node_modules/@ngtools/webpack/src/transformers/ast_helpers.js:131:59)
    at Object.<anonymous> (/usr/local/google/home/stephenfluin/workspace/test/devfest/node_modules/@ngtools/webpack/src/transformers/ast_helpers.js:47:20)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/usr/local/google/home/stephenfluin/workspace/test/devfest/node_modules/@ngtools/webpack/src/transformers/index.js:14:10)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
```

## 🔬 Minimal Reproduction
git clone git@github.com:GDGTC/devfest.git
cd devfest
yarn
ng serve # works
ng update @angular/cli@8 @angular/core@8 --force
ng serve # still works
ng update @angular/cli@9 @angular/core@9 --force --allow-dirty
ng serve # failure

## 🌍 Your Environment
<pre><code>
Angular CLI: 9.0.0
Node: 12.14.1
OS: linux x64

Angular: 9.0.0
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... platform-server, router, service-worker
Ivy Workspace: <error>

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.900.0
@angular-devkit/build-angular     0.900.0
@angular-devkit/build-optimizer   0.900.0
@angular-devkit/build-webpack     0.900.0
@angular-devkit/core              9.0.0
@angular-devkit/schematics        9.0.0
@angular/cdk                      8.0.0-rc.1
@angular/fire                     5.2.1
@angular/material                 8.0.0-rc.1
@ngtools/webpack                  9.0.0
@schematics/angular               9.0.0
@schematics/update                0.900.0
rxjs                              6.5.4
typescript                        3.7.5
webpack                           4.41.2

</code></pre>

**Comment by jsaguet** [comment]
I am encountering the same issue with an ionic app.
I've noticed that angular/cli v9-rc.13 was working fine but rc.14 then final version 9 threw me the exact same error.
Let me know if I you need mode information

**Comment by filipesilva** [comment]
Heya, looking into this atm, I believe it's due to a change in https://github.com/angular/angular-cli/pull/16822.

**Comment by eun-choi** [comment]
Same for me. 
Build succeeds until version 9.0.0-rc.12
But, build failes above version 9.0.0-rc.13 and 9.0.0

**Comment by filipesilva** [comment]
Fix incoming in https://github.com/angular/angular-cli/pull/16859

**Comment by mgechev** [comment]
Fixed by https://github.com/angular/angular-cli/pull/16859 and released in v9.0.1.
